diff --git a/dist/index.js b/dist/index.js
index 914285807221289f4a9b822fbcf18bec8e6da588..f5a99ea0e6a9d79fb30c9e631a6b0f6dca28c290 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,4 +1,4 @@
-import { getHighlighter } from 'shiki';
+import { getSingletonHighlighter as getHighlighter } from 'shiki';
 import { visit } from 'unist-util-visit';
 import { toString } from 'hast-util-to-string';
 import rangeParser2 from 'parse-numeric-range';
